addon/components/polaris-drop-zone.js

```diff
@@ -4,6 +4,7 @@ import { or } from '@ember/object/computed';
 import { classify } from '@ember/string';
 import { throttle, scheduleOnce } from '@ember/runloop';
 import { isNone, isPresent } from '@ember/utils';
+import { createUniqueIDFactory } from '../utils/create-unique-id-factory';
```

Comment:
We've eschewed this in the past in favour of using Ember's unique IDs via `guidFor(this)`, is there any reason not to continue that path here?

Reply:
Yeah I know, just that so far we've avoided doing that and I'd like to keep our addon self-consistent. We can always look at switching all of the `guidFor`s across to follow the React implementation at a later stage :+1

Reply:
yeah, I think this is not really a problem, but more an advantage that we use Ember 😉